src/main/java/pl/javastart/task/logic/MixPhoneContract.java

```diff
@@ -0,0 +1,83 @@
+package pl.javastart.task.logic;
+
+public class MixPhoneContract extends Contract {
```

Comment:
Ta klasa się u mnie nie kompiluje `Class 'MixPhoneContract' must either be declared abstract or implement abstract method 'contractName()' in 'Contract'`